cluster/manifests/coredns-local/daemonset-coredns.yaml

```diff
@@ -51,6 +51,18 @@ spec:
         - containerPort: 53
           name: dns-tcp
           protocol: TCP
+{{- if eq .Cluster.ConfigItems.dns_unbound_liveness_probe "true" }}
+        livenessProbe:
+          exec:
+            command:
+            - dig
+            - "+short"
+            - "@127.0.0.1"
+            - "kubernetes.default.svc.cluster.local"
```

Comment:
I'd use an external hostname here.

Reply:
On the other hand, this will cause all containers to restart if there's a hiccup with AWS DNS, so maybe an internal one would work better.

Reply:
Yeah, this is also what I worry about. We just want to test that the unbound cache is working, not the whole DNS infra.